packages/taro-router/rollup.config-1745118866046.mjs

```
@@ -0,0 +1,51 @@
+            file: 'dist/index.cjs.js',
+            format: 'cjs',
+        }
+    },
+    {
+        output: {
+            file: 'dist/index.esm.js',
+            format: 'es',
+        }
+    }];
+var rollup_config = defineConfig(variesConfig.map(v => {
+    return Object.assign({}, baseConfig, v);
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(@typescript-eslint/indent)